src/domain/bridge/contracts/decoders/lifi-diamond-decoder.helper.spec.ts

```diff
@@ -0,0 +1 @@
+it.todo('LiFiDiamondDecoder');
```

Comment:
As we currently don't know the exact methods we will need to decode, I've intentionally left this blank to tackle later.